src/main/resources/mainnet.conf

```diff
@@ -34,6 +34,9 @@ ergo {
 
     # List with hex-encoded identifiers of transactions banned from getting into memory pool
     blacklistedTransactions = []
+
+    # maximum cost of transaction for it to be propagated
+    maxTransactionCost = 5000000
```

Comment:
Looks too high.

Reply:
Block limit is 7M, and 5M is non-discriminative to most known existing transactions while also limiting them.